### DISCUSSION TITLE
Onboarding & Operations Discussion Topics

### DISCUSSION THREAD

**Discussion by MauriceReeves-usds** [CDCgov/prime-reportstream#3685] [General]
Let's collect discussion topics for office hours and training here. If you want to discuss something, or learn more about something, put it here.

**Comment by anshulkumar-usds** [comment]
Ideas from team retro on 1/4:
1. Setup VPN access
2. Review new mappers
3. Resubmission of files to STLTs (support queue) - **1/7/22 Brick Green**
4. API Sender Onboarding overview (tech demo)
5. STLT Onboarding overview (tech demo)
6. How to setup and configure a download site for STLTs (including OKTA setup)
7. Token based authentication (tech demo)
8. Review of new and old mappers (probably some I don't realize exist)
9. review of current strategy for expanding beyond COVID (has that started for HCA?)

**Comment by MauriceReeves-usds** [comment]
I'd like to discuss the technical debt that is out there too.

**Comment by brick-green** [comment]
- Review of new and old mappers (probably some I don't realize exist)
- review of current strategy for expanding beyond COVID (has that started for HCA?)

**Comment by brick-green** [comment]
Zenhub labels - I see a few labels that I am unfamiliar with and would like to learn what the expectations are for using labels

@Adrian-Brewster can you add this discussion topic to the list?
